app/experimenter/nimbus-ui/src/components/GQLErrorAlert/index.tsx

```diff
@@ -0,0 +1,90 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { ApolloError, ServerError, ServerParseError } from "@apollo/client";
+import { GraphQLError } from "graphql";
+import React from "react";
+import { Alert } from "react-bootstrap";
+import sentryMetrics from "../../services/sentry";
+
+type NetworkError = ApolloError["networkError"];
+
+// Type guards - If the `networkError` is a `ServerError`, we want to display its `result.errors`
+// array containing extra info
+function isNetworkServerError(
+  networkError: NetworkError,
+): networkError is ServerError {
+  return (networkError as ServerError)?.result !== undefined;
+}
+
+// If the `networkError` is a `ServerParseError`, we want to display `bodyText` and `statusCode`
+function isNetworkServerParseError(
+  networkError: NetworkError,
+): networkError is ServerParseError {
+  return (
+    (networkError as ServerParseError)?.bodyText !== undefined &&
+    (networkError as ServerParseError)?.statusCode !== undefined
+  );
+}
+
+export const GQLErrorAlert = ({ error }: { error: ApolloError }) => {
+  // without this, we only have access to error.message
+  const allErrors = JSON.parse(JSON.stringify(error));
+  const { graphQLErrors = [] } = allErrors as ApolloError;
+  const { networkError } = allErrors as ApolloError;
+  const networkServerErrors =
+    networkError && isNetworkServerError(networkError)
+      ? networkError.result.errors
+      : [];
+
+  // we want to report Errors and NetworkErrors
+  // graphQLErrors should be captured on the server-side
+  if (networkError || error instanceof Error) {
+    sentryMetrics.captureException(error);
```

Comment:
Prior to this change, if we [caught the error and returned anything](https://github.com/mozilla/experimenter/pull/5362/files#diff-77ef9304606052b73cebb4b85f40b658b05bed9a51065504a32148a563aa0de7R25), it wasn't reported to Sentry since it didn't bubble up to `AppErrorBoundary` any longer.
 
This seemed right to me, followed some of what we laid out in [`fxa-settings` docs](https://github.com/mozilla/fxa/tree/main/packages/fxa-settings#gql-error-handling). Let me know otherwise. 😎